functions/Set-DbaLogin.ps1

```diff
@@ -253,8 +270,20 @@ function Set-DbaLogin {
                 # Change the password
                 if (Test-bound -ParameterName 'SecurePassword') {
                     try {
+                        # these settings are required for must_change and are validated above
+                        if (Test-Bound MustChange) {
+                            Write-Message -Message "Setting check_policy and check_expiration to true for $l because the must_change option requires it." -Level Warning
```

Comment:
This is not the purpose of providing MustChange in the command the way I read it. Enabling these settings breaks the use of that parameter's purpose. It is meant to be used in conjunction with a new password being provided so it will force a change in the password.

Out intention in that workflow is not to enforce any policies; having the password policy enforced can cause the password change itself to fail if they are using a weak password.